crates/js-component-bindgen/src/ts_bindgen.rs

```diff
@@ -154,6 +199,14 @@ pub fn ts_bindgen(
                         (iface_id.as_ref(), iface)
                     }
                 };
+
+                if !feature_gate_allowed(resolve, package, &stability)
+                    .expect("failed to check feature gate for export")
```

Comment:
Instead of a panic, could this be an error?

Reply:
Ah so the reason this is an `expect` is because `ts_bindgen()` doesn't return a result -- I'll change the signature here and update the rest.

- [ ] change `ts_bindgen` to return a `Result`